pgjdbc/src/main/java/org/postgresql/jdbc/PgConnection.java

```diff
@@ -733,6 +733,7 @@ private void initObjectTypes(Properties info) throws SQLException {
   /**
    * <B>Note:</B> even though {@code Statement} is automatically closed when it is garbage
    * collected, it is better to close it explicitly to lower resource consumption.
+   * The spec says that calling close on a closed connection is a noop
```

Comment:
You are missing a period at the end of this sentence. I'd also write it as "no-op", personally.